bobtask/artifact.go

```diff
@@ -170,11 +171,62 @@ func (t *Task) ArtifactPack(artifactName hash.In) (err error) {
 	return nil
 }
 
+func (t *Task) pathTargets() ([]string, error) {
+	targets := []string{}
+	for _, path := range t.target.Paths {
+		stat, err := os.Stat(filepath.Join(t.dir, path))
+		if err != nil {
+			return targets, err
+		}
+
+		if stat.IsDir() {
+			// TODO: Read all files from dir.
+			root := filepath.Join(t.dir, path)
+			_ = filepath.WalkDir(root, func(path string, d fs.DirEntry, err error) error {
+				if err != nil {
+					return err
+				}
+				if d.IsDir() {
+					return nil
+				}
+
+				targets = append(targets, path)
+				return nil
+			})
+		} else {
+			targets = append(targets, filepath.Join(t.dir, path))
+		}
+
+	}
+	return targets, nil
+}
+
+// saveDockerImageTargets calls `docker save` and returns a path to the tar archive.
+func (t *Task) saveDockerImageTargets() ([]string, error) {
+	targets := []string{}
+
+	// TODO: change this path based implementation to docker tag
```

Comment:
TODO, is this the right syntax? Not sure if we should reuse the `path` directive.